jwst/stpipe/pipeline.py

```diff
@@ -33,19 +33,30 @@
 
 from __future__ import absolute_import, division, print_function
 
-from os.path import dirname, join
 import gc
+from os.path import dirname, join, split, splitext
+import re
 
 from .configobj.configobj import Section
 
 from . import config_parser
 from . import Step
 
+REMOVE_SUFFIX = '(.+?)(_(rate|cal)(ints)?)?$'
+
 
 class Pipeline(Step):
     """
     A Pipeline is a way of combining a number of steps together.
     """
+
+    # Configuration
+    spec = """
+    output_basename = string(default=None) # Output base name
+    output_ext = string(default=".fits") # Output extension
+    suffix = string(default=None) # Suffix for output file name
+    output_use_model = boolean(default=False)
```

Comment:
What does this `output_use_model` config option do?  Would be good to have a comment documenting it.